### ISSUE TITLE
MultiNEAT is not automatically installed

### ISSUE THREAD

**Issue by DaanZ** [ci-group/revolve#89]
We should supply the revolve CMakeLists.txt with 

add_subdirectory(thirdparty/MultiNEAT)

But this does not build MultiNEAT successfully.

```
CMake Error at build/thirdparty/MultiNEAT/MultiNEATConfig.cmake:21 (include):
  include could not find load file:
/home/daanz/revolve_workspace/revolve_battery/build/thirdparty/MultiNEAT/MultiNEATTargets.cmake
Call Stack (most recent call first):
  cpprevolve/revolve/brains/CMakeLists.txt:21 (find_package)
```

The MultiNEAT cmake file should create the MultiNEATTargets.cmake, but this is not the case.

**Comment by portaloffreedom** [comment]
MultiNEAT is supposed to be installed system wise to make this work.
`MultiNEATConfig.cmake` is generated depending on your configuration.

This is the current situation, I understand that there is a better way to do this, without installing MultiNEAT and it would be preferrable, but I need more experience on cmake to understand how to do this properly.

**Comment by DaanZ** [comment]
I am going to read https://zhjwpku.com/assets/pdf/books/Managing.Projects.With.Gnu.Make.3Rd.Edition.pdf to possibly assist with this.

**Comment by portaloffreedom** [comment]
Not really. Make is an old tool very manual and unpractical. What I used as a reference in modern cmake is this: https://pabloariasal.github.io/2018/02/19/its-time-to-do-cmake-right/
